js/bundle/src/cli.ts

```diff
@@ -68,6 +98,10 @@ function readOptions() {
       'manifest URL (only valid with format version "' + B1 + '")'
     )
     .option('-o, --output <file>', 'webbundle output file', 'out.wbn')
+    .option(
+      '-h, --headerOverride <jsonFilePath>',
+      'path to a JSON file specifying the headers'
```

Comment:
nit: Can you also specify the format of the JSON file here? I.e. that it must be an object where keys are header names and values are strings representing the header values?